mssql-2019/docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: '3.7'
+services:
+  mssql:
+    image: mcr.microsoft.com/mssql/server:2019-latest
+    environment: 
+      - SA_PASSWORD=Dev12345
```

Comment:
Since there is a minimum 8 characters requirements for the SQL Server database, I get it that we cannot keep this password the same as others. i.e., `Dev1234`
What do you think about changing [Postgres](https://github.com/tidalmigrations/db-scripts/blob/main/postgres-10-5/sql/tidal_setup.sql#L1) and [MySQL](https://github.com/tidalmigrations/db-scripts/blob/main/mysql-8/sql/tidal_setup.sql#L1)'s password to this one? i.e., `Dev12345`

Reply:
Sure thing, for consistency. Keep in mind the password for the tidal tools user is still `Dev1234` as there's no limit for that, but yeah I wouldn't mind having it uniform across everything in the next PR.

Reply:
Good idea, lets make sure we keep those password consistent 👍